### ISSUE TITLE
Builds break after failing to install Quicklisp on Allegro

### ISSUE THREAD

**Issue by DalekBaldwin** [luismbo/cl-travis#12]
I'm not sure whether the blame lies with cl-travis or the latest Quicklisp update (the client was updated April 16 and again April 24) or the interaction between the two, but this problem has caused builds to fail on Allegro since April 22 at the latest. Example: https://travis-ci.org/DalekBaldwin/check-it/jobs/61141813

```
Installing quicklisp.
Failed to install quicklisp.
can't read-byte from a string stream: #<EXCL:STRING-INPUT-SIMPLE-STREAM ";;  -*- mode: c" pos 0 @ #x206a8a3a>An error occured while processing command line arguments
The command "if [ -x ./install.sh ] && head -2 ./install.sh | grep '^# cl-travis' > /dev/null; then ./install.sh; else curl https://raw.githubusercontent.com/luismbo/cl-travis/master/install.sh | sh; fi" failed and exited with 1 during .
Your build has been stopped.
```

**Comment by luismbo** [comment]
Re-triggered the latest cl-travis build to see if I can reproduce it: https://travis-ci.org/luismbo/cl-travis/builds/46838166

**Comment by luismbo** [comment]
Reproduced: https://travis-ci.org/luismbo/cl-travis/jobs/46838168

@DalekBaldwin I believe this is a CIM bug. Pinging @KeenS: can you confirm this?

**Comment by KeenS** [comment]
@DalekBaldwin @luismbo verified. Since it is midnight at Japan, I'll fix it tomorrow morning.

**Comment by luismbo** [comment]
@KeenS no problem. :) Thanks!

**Comment by KeenS** [comment]
@DalekBaldwin @luismbo I think it got fixed.

**Comment by DalekBaldwin** [comment]
My stuff seems to work now. Thanks!

**Comment by luismbo** [comment]
@KeenS great, thanks! BTW, is there a way to grab and install a specific version of CIM? My goal would be to only push a new version to cl-travis users after it passed cl-travis's test suite.

**Comment by KeenS** [comment]
@luismbo There exists an environment variable `CIM_INSTALLl_BRANCH`.
You can use it to specify install branch. Like this:

```
curl -L https://raw.github.com/KeenS/CIM/master/scripts/cim_installer | CIM_INSTALL_BRANCH=develop CIM_HOME=~/tmp/cim /bin/sh
```

. This isn't documented but public interface.

Now I *found* you can put committish to it. Like below:

```
curl -L https://raw.github.com/KeenS/CIM/master/scripts/cim_installer | CIM_INSTALL_BRANCH=0ccbe2eba01692682905e946c7a0074d1f2d9b03 CIM_HOME=~/tmp/cim /bin/sh
```

. I didn't intended to implement this behavior but realized by GitHub's consistent API design. However as you know GitHub is far stable than CIM, I can say you can rely on this behavior.

**Comment by luismbo** [comment]
That's neat @KeenS! Thanks. I will implement this in cl-travis.

**Comment by luismbo** [comment]
Implemented in 2dd595e7c5481d355f3587d497b0227d6bd52baa.
